.Gruntfile.babel.js

```diff
@@ -118,7 +118,7 @@ module.exports = (grunt) => {
         cmd: './node_modules/.bin/mocha --require Gruntfile.js --exit -R spec --bail "{./{,!(node_modules)/**/}*.test.js,./test/*.js}"',
         options: {env: {LOAD_NO_FILE: 'true', ...process.env}}
       },
-      standard: './node_modules/.bin/standard "**/*.{js,vue}"',
+      standard: './node_modules/.bin/standard "**/*.{js,vue}" --fix', // TODO don't forget to remove --fix
```

Comment:
If you want it so bad you could just add a `standardfix` command :P

Reply:
Since the first day I don this, I can live with it, actually I never fully understood why can't he have it be default. It helps so damn much while developing O.o

Reply:
Because `standard` gets run every time `test` is called, and we don't want any code modifications to happen unexpectedly, they have to be deliberate.

Reply:
If we have linter errors, `grunt test` won't run at all (the linter will throw an error before).

The linter only changes code that is against the rules (warnings or errors). If we want to ignore the linter in a specific line we can use `// eslint-disable-line`.
When we don't do it, the error will persist and 99% of the times it was because of an extra `;` or missing space - so I don't see any disadvantages with `standardfix` so far 🤔

Reply:
> If we have linter errors, grunt test won't run at all (the linter will throw an error before).

That's exactly the behavior we want from `grunt test`.

> so I don't see any disadvantages with standardfix so far 

Do you mean you don't see any advantages?

Anyway, we can't have `grunt test` resulting in code being modified. It's potentially dangerous, and similarly, we can't have `grunt dev` resulting in code changes.

This needs to be removed before I can merge this, and if you'd like it here, the best way to get it in is to add it via a `standardfix` command. Then you'd run `grunt exec:standardfix`.

Reply:
No, I mean _ disadvantages_. But it's okay, I've removed the fix as expected, I left it there just to help me out during development :)